src/index.ts

```diff
@@ -114,23 +122,82 @@ async function run() {
     core.endGroup();
 
     core.startGroup("Compute gas diff");
-    const diffRows = computeProgramDiffs(referenceReports.programs, compareReports.programs);
-    core.info(`Format markdown of ${diffRows.length} diffs`);
+    const [diffCircuitRows, diffBrilligRows] = computeProgramDiffs(
+      referenceReports.programs,
+      compareReports.programs
+    );
+
+    // NOTE: This is hacky but we are assuming that if a workspace report was generated using
+    // `--force-brillig` then we should have a single unconstrained function marked with "main".
+    // If we compiled our entire workspace to Brillig then we can generate a Brillig report instead of a circuit report.
+    let brillig_report = false;
+    if (
+      referenceReports.programs.length > 0 &&
+      referenceReports.programs[0].unconstrained_functions.length === 1 &&
+      referenceReports.programs[0].unconstrained_functions[0].name === "main"
+    ) {
+      brillig_report = true;
+    }
```

Comment:
Here we're assuming that if the first project in the workspace is unconstrained then we want an unconstrained report, no? This seems kinda fragile if we're reliant on the ordering of projects within a mixed workspace. Why not output the two reports separately similarly to how we do for shell/markdown?

Reply:
done